modules/virt-understanding-smart-cloning.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc
+
+[id="virt-understanding-smart-cloning_{context}"]
+= Understanding smart cloning
```

Comment:
Although it is good to provide some initial context in the assembly, I think that you could put some or all of what you had as the assembly intro into this topic, which is learning about smart cloning.

Unless you really only want to include the process of what happens, then perhaps you could consider a clarification to the heading, like "Understanding the smart cloning process" or something?

Reply:
I moved the smart-cloning specific content into the contextual module. Good call. The assembly only talks about cloning (briefly).